packages/jsii-pacmak/lib/targets/js.ts

```diff
@@ -2,26 +2,38 @@ import * as spec from '@jsii/spec';
 
 import { Generator } from '../generator';
 import { PackageInfo, Target } from '../target';
+import { toReleaseVersion } from './version-utils';
+
+import { TargetName } from '.';
 
 export default class JavaScript extends Target {
   public static toPackageInfos(
     assm: spec.Assembly,
   ): { [language: string]: PackageInfo } {
     const packageInfo: PackageInfo = {
       repository: 'NPM',
-      url: `https://www.npmjs.com/package/${assm.name}/v/${assm.version}`,
+      url: `https://www.npmjs.com/package/${assm.name}/v/${toReleaseVersion(
+        assm.version,
+        TargetName.JAVASCRIPT,
+      )}`,
       usage: {
         'package.json': {
           language: 'js',
           code: JSON.stringify({ [assm.name]: `^${assm.version}` }, null, 2),
```

Comment:
just checking - need to call `toReleaseVersion()` here?

Reply:
I guess yes (for consistency). But `toReleaseVersion` is basically no-op for JavaScript.